queries/userloc.py

```diff
@@ -56,6 +56,7 @@
 
 QUserLocationCurrent →
     "hvar" "er" "ég" QULocEiginlega? QULocLocated? QULocInTheWorld? QULocNow?
+    | "hvar" QULocInTheWorld? "er" "ég" QULocEiginlega QULocLocated? QULocNow?
```

Comment:
The rules in lines 58 and 59 are a bit suspicious-looking - they might be ambiguous, i.e. both would match the same string. That is perhaps not a big problem but we generally try to avoid explicit ambiguity.